source/MaterialXRuntime/RtConnectableApi.cpp

```diff
@@ -0,0 +1,49 @@
+//
+// TM & (c) 2019 Lucasfilm Entertainment Company Ltd. and Lucasfilm Ltd.
+// All rights reserved.  See LICENSE.txt for license.
+//
+
+#include <MaterialXRuntime/RtConnectableApi.h>
+#include <MaterialXRuntime/RtPrim.h>
+
+namespace MaterialX
+{
+
+namespace
+{
+    RtTokenMap<RtConnectableApiPtr> s_connectableApiRegistry;
+}
+
+bool RtConnectableApi::acceptConnection(const RtInput& input, const RtOutput& output) const
```

Comment:
The default implementation just checks if the attribute types are matching, so same as the old behavior.